db/seeds/questionnaires/student_post_assessment.rb

```diff
@@ -1,259 +1,130 @@
-                                 'Je bent fantastisch op weg! Ga zo door. LINK'],
-                       section_end: true
-                     }, {
-                       section_start: 'Missen van vragenlijsten',
-                       id: :v35,
-                       type: :checkbox,
-                       title: 'Wat waren de redenen dat je wel eens een vragenlijst hebt gemist? (meerdere antwoorden mogelijk)',
-                       options: ['Ik heb nooit een vragenlijst gemist',
-                                 'Ik kreeg geen sms',
-                                 'De link naar de vragenlijst werkte niet',
-                                 'Ik had geen tijd',
-                                 'Ik had geen zin',
-                                 'Ik was het vergeten',
-                                 'Mijn batterij was leeg',
-                                 'Ik zat op dat moment niet met mijn telefoon op wifi',
-                                 'De databundel van mijn telefoon was op',
-                                 'De vragenlijst was al verlopen'
-                       ]
-                     }, {
-                       id: :v36,
-                       type: :radio,
-                       title: 'Zou je het erg vinden als jouw begeleider op de hoogte wordt gesteld als jij twee of meer metingen mist?',
-                       options: ['Ja', 'Nee'],
-                       section_end: true
-                     }, {
-                       section_start: 'Beloning',
-                       id: :v37,
-                       type: :radio,
-                       title: 'Vind je dat je genoeg beloning krijgt voor wat je moet doen?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, voor het werk dat ik heb gedaan zou ik dit een eerlijke beloning vinden: €'
-                     }, {
-                       id: :v38,
-                       type: :radio,
-                       title: 'Je hebt nu drie weken meegedaan aan dit onderzoek. Denk je dat je ditzelfde onderzoek ook voor zeven maanden zou volhouden voor €70?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, omdat:',
-                       section_end: true
-                     }, {
-                       section_start: 'Tot slot',
-                       id: :v39,
-                       type: :textarea,
-                       title: 'Wat zou jij willen verbeteren aan de webapp die je de afgelopen drie weken hebt gebruikt?',
-                       section_end: true
-                     }]
+  id: :v1,
+  type: :radio,
+  title: 'Ben je definitief gestopt met je opleding (zonder diploma)?',
+  options: [
+    { title: 'Ja', shows_questions: %i[v2 v3 v4] },
+    'Nee'
+  ],
+  show_otherwise: false
+}, {
+  id: :v2,
+  hidden: true,
+  type: :date,
+  max: [2018, 11, 01],
+  required: true,
```

Comment:
Niet vergeten even te testen dat het submitten van het form goedgaat als we deze vraag hidden laten. In principe zou het goed moeten gaan maar je weet maar nooit met een nieuw inputelement (zeker omdat er een boel javascript achter hangt).